build/configs/rtl8730e/audio/defconfig

```diff
@@ -0,0 +1,1570 @@
+# CONFIG_UART0_IFLOWCONTROL is not set
+# CONFIG_UART0_OFLOWCONTROL is not set
+# CONFIG_UART_SERIALDRIVER is not set
+# CONFIG_UART0_SERIALDRIVER is not set
+# CONFIG_UART1_SERIALDRIVER is not set
+# CONFIG_UART2_SERIALDRIVER is not set
+# CONFIG_UART3_SERIALDRIVER is not set
+# CONFIG_UART4_SERIALDRIVER is not set
+# CONFIG_UART5_SERIALDRIVER is not set
+# CONFIG_UART6_SERIALDRIVER is not set
+# CONFIG_UART7_SERIALDRIVER is not set
+# CONFIG_UART8_SERIALDRIVER is not set
+CONFIG_UART1_RXBUFSIZE=1024
+CONFIG_UART1_TXBUFSIZE=1024
+CONFIG_UART1_BAUD=115200
+CONFIG_UART1_BITS=8
+CONFIG_UART1_PARITY=0
+CONFIG_UART1_2STOP=0
+# CONFIG_UART1_IFLOWCONTROL is not set
+# CONFIG_UART1_OFLOWCONTROL is not set
+CONFIG_UART4_RXBUFSIZE=1024
+CONFIG_UART4_TXBUFSIZE=1024
+CONFIG_UART4_BAUD=1500000
+CONFIG_UART4_BITS=8
+CONFIG_UART4_PARITY=0
+CONFIG_UART4_2STOP=0
+# CONFIG_UART4_IFLOWCONTROL is not set
+# CONFIG_UART4_OFLOWCONTROL is not set
+# CONFIG_LPUART_SERIALDRIVER is not set
+# CONFIG_LPUART0_SERIALDRIVER is not set
+# CONFIG_LPUART1_SERIALDRIVER is not set
+# CONFIG_LPUART2_SERIALDRIVER is not set
+# CONFIG_LPUART3_SERIALDRIVER is not set
+# CONFIG_LPUART4_SERIALDRIVER is not set
+# CONFIG_LPUART5_SERIALDRIVER is not set
+# CONFIG_LPUART6_SERIALDRIVER is not set
+# CONFIG_LPUART7_SERIALDRIVER is not set
+# CONFIG_LPUART8_SERIALDRIVER is not set
+
+#
+# UART1 Configuration
+#
+
+#
+# UART4 Configuration
+#
+# CONFIG_SENSOR is not set
+# CONFIG_USBDEV is not set
+# CONFIG_USBHOST is not set
+# CONFIG_FOTA_DRIVER is not set
+
+#
+# System Logging
+#
+# CONFIG_RAMLOG is not set
+# CONFIG_SYSLOG_CONSOLE is not set
+
+#
+# T-trace
+#
+# CONFIG_TTRACE is not set
+# CONFIG_IOTDEV is not set
+
+#
+# Wireless Device Options
+#
+CONFIG_DRIVERS_WIRELESS=y
+CONFIG_DRIVERS_BLUETOOTH=y
+# CONFIG_DRIVERS_BR_EDR is not set
+CONFIG_DRIVERS_BLE=y
+# CONFIG_VIRTUAL_BLE is not set
+CONFIG_RTK_BLE=y
+# CONFIG_OTP is not set
+CONFIG_SECURITY_LINK_DRV=y
+CONFIG_SECURITY_LINK=y
+
+#
+# Networking Support
+#
+CONFIG_ARCH_HAVE_NET=y
+# CONFIG_ARCH_HAVE_PHY is not set
+CONFIG_NET=y
+CONFIG_NET_LWIP=y
+
+#
+# LwIP options
+#
+CONFIG_NET_IPv4=y
+CONFIG_NET_IP_DEFAULT_TTL=255
+# CONFIG_NET_IP_FORWARD is not set
+CONFIG_NET_IP_OPTIONS_ALLOWED=y
+CONFIG_NET_IP_FRAG=y
+CONFIG_NET_IP_REASSEMBLY=y
+CONFIG_NET_IPV4_REASS_MAX_PBUFS=60
+CONFIG_NET_IPV4_REASS_MAXAGE=5
+CONFIG_NET_ICMP=y
+CONFIG_NET_ICMP_TTL=255
+# CONFIG_NET_BROADCAST_PING is not set
+# CONFIG_NET_MULTICAST_PING4 is not set
+CONFIG_NET_LWIP_IGMP=y
+CONFIG_NET_LWIP_MEMP_NUM_IGMP_GROUP=8
+CONFIG_NET_ARP=y
+CONFIG_NET_ARP_TABLESIZE=10
+CONFIG_NET_ARP_QUEUEING=y
+CONFIG_NET_ETHARP_TRUST_IP_MAC=y
+CONFIG_NET_ETH_PAD_SIZE=0
+# CONFIG_NET_ARP_STATIC_ENTRIES is not set
+CONFIG_NET_UDP=y
+# CONFIG_NET_NETBUF_RECVINFO is not set
+CONFIG_NET_UDP_TTL=255
+CONFIG_NET_UDPLITE=y
+CONFIG_NET_TCP=y
+CONFIG_NET_TCP_TTL=255
+CONFIG_NET_TCP_WND=26280
+# CONFIG_NET_WND_SCALE is not set
+CONFIG_NET_TCP_MAXRTX=12
+CONFIG_NET_TCP_SYNMAXRTX=6
```

Comment:
Both should be 7. @hk-gwak Could you confirm this?

Reply:
Both value should be 7 or greater than 7. So only CONFIG_NET_TCP_SYNMAXRTX=6 needs to be fixed to 7.
```suggestion
CONFIG_NET_TCP_SYNMAXRTX=7
```

Reply:
Both value should be 7 or greater than 7. So only CONFIG_NET_TCP_SYNMAXRTX=6 needs to be fixed to 7.
```suggestion
CONFIG_NET_TCP_MAXRTX=12
CONFIG_NET_TCP_SYNMAXRTX=7
```